src/js/player.js

```diff
@@ -1168,13 +1168,9 @@ Player.prototype.selectSource = function(sources){
  * @return {String} The current video source when getting
  * @return {String} The player when setting
  */
-Player.prototype.src = function(source){
+Player.prototype.src = function(source=this.techGet('src')){
```

Comment:
You can do this?? And it will execute each time the function is called, not just when the function is created?